apis/nucleus/src/viz.js

```diff
@@ -135,6 +136,13 @@ export default function viz({
             path: cellRef.current.getHypercubePath(),
           };
 
+          if (
```

Comment:
I would think `if (typeof extensionDefinition.definition?.dataHandler === 'function')` is sufficient?